website/static/worker.js

```diff
@@ -12,34 +12,34 @@ function importScriptOnce(url) {
 
 // this is required to only load parsers when we need them
 var parsers = {
-  // JS - Babylon
-  get babylon() {
+  // JS - Babel
+  get babel() {
     importScriptOnce("lib/parser-babylon.js");
-    return prettierPlugins.babylon.parsers.babylon;
+    return prettierPlugins.babel.parsers.babel;
```

Comment:
`prettierPlugins.<parser-filename>.parsers.<parser-name>`

```suggestion
    return prettierPlugins.babylon.parsers.babel;
```